### ISSUE TITLE
Module no longer works with latest updates to Foundry or DnD5e

### ISSUE THREAD

**Issue by GambetTV** [illandril/FoundryVTT-hotbar-uses#34]
With no other modules active, this module does not seem to show the number of uses spells or ammunition have left in the hotbar. I am using the latest version of Foundry and DnD5e. I am not exactly sure when this stopped working as it's mostly my players who use this module, but it's probably recent because tonight's the first time I've heard about it in my weekly game.

**Comment by illandril** [comment]
I cannot reproduce this issue using DnD5e version 1.6.1, Foundry version 9 build 269, and Illandril's Hotbar Uses 2.4.1..

Try clearing your browser's cache and trying again.

If it still doesn't work, try uninstalling and re-installing the module (fully uninstall it and re-install it from the Foundy Configuration and Setup page, don't just remove it and re-add it to your world).

If it still doesn't work, open up your browser's Developer Tools (press F12) and check the console for any errors, and post those errors here or send them to me over Discord (if you're using Chrome or the standalone Foundry application, you can right-click in the Console and select "Save As" to easily save the entire console log).

**Comment by GambetTV** [comment]
When I only have your module installed, I don't get any console errors except one that I assume is unrelated as it's about Helvetica not being available. I went through a few of my player's accounts and noticed one of them has a hotbar with a macro for a spell which still has the number of spell uses left. However, when I try to drag that same exact spell into another slot, the macro shows up but the number of spells does not.

This bug is affecting players using Chrome and Firefox, and me using the Foundry app. I dropped the console log for you anyway. Not really sure what the issue is. I tried everything else you suggested, and even on reinstalling it, no change.

[log.txt](https://github.com/illandril/FoundryVTT-hotbar-uses/files/8735582/log.txt)
